tests/driver_nrfmin/Makefile

```diff
@@ -4,7 +4,7 @@ include ../Makefile.tests_common
 # include nrfmin, the main purpose of this test
 USEMODULE += nrfmin
 # use a minimal GNRC configuration
-USEMODULE += gnrc_netdev_default
+USEMODULE += netdev_default
```

Comment:
Given the comment, I think the line should be moved up. But I don't believe the module is needed at all, as `nrfmin` is included above.